lib/screens/addLocation.js

```diff
@@ -0,0 +1,78 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, View, ScrollView, Button } from 'react-native';
+import Dropdown from './dropdown';
+
+export default class AddLocation extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ViewArray: []
+    }
+    this.Array_Value_Index = 2;
+  }
+  Add_New_View_Function = () => {
+    let New_Added_View_Value = { Array_Value_Index: this.Array_Value_Index }
+    this.setState({ ViewArray: [...this.state.ViewArray, New_Added_View_Value] }, () => {
+      this.Array_Value_Index++;
+    });
+  }
+  render() {
+    let Render_Animated_View = this.state.ViewArray.map((item, key) => {
```

Comment:
For lambda functions, if there's only a return statement in the function, you can skip the curly braces + return and directly point the arrow to what you want to return
Eg.
```
() => {
  return 5;
}
```
is the same thing as 
`() => 5`

So in this case we can wrap the entire jsx here in parenthesis and get rid of the return